vmdb/app/models/ems_refresh/save_inventory_orchestration_stacks.rb

```diff
@@ -15,11 +15,18 @@ def save_orchestration_templates_inventory(_ems, hashes, _target = nil)
       hashes.zip(templates).each { |hash, template| hash[:id] = template.id }
     end
 
+    def all_orchestration_stacks(ems)
```

Comment:
no need for this method. now `orchestration_stacks_with_nested` is available for all emses